README.md

```diff
@@ -108,7 +108,56 @@ Migration guide:
 This function is deprecated. Use the `torch.nn.attention.sdpa_kernel` context manager instead.
 
 Migration guide:
+
 Each boolean input parameter (defaulting to true unless specified) of `sdp_kernel` corresponds to a `SDPBackened`. If the input parameter is true, the corresponding backend should be added to the input list of `sdpa_kernel`.
 
+### TOR102 Unsafe use of function 
+
+#### torch.load
+
+The use of `torch.load` without the `weights_only` parameter is unsafe. Loading an untrusted pickle file may lead to the execution of arbitrary malicious code and potential security issues.
+
+Migration Guide:
+
+Explicitly set `weights_only=False` only if you trust the data you load and full pickle functionality is needed, 
+
+```
+torch.load('<path_to_file>', weights_only=False) 
+```
+otherwise set `weights_only=True`.     
+```
+torch.load('<path_to_file>', weights_only=True)
+```
+
+### TOR104 Use of non-public function
+
+#### torch.utils.data._utils.collate.default_collate
+
+Public functions are well-documented and supported by the library maintainers and the use of the non-public function `torch.utils.data._utils.collate.default_collate` is discouraged as it can can change without notice in future versions, leading to potential breakage in your code.   
+
+Migration Guide:
+
+For better maintainability and compatibility, please use the public function `torch.utils.data.dataloader.default_collate` instead.
+
+### TOR201 Use of deprecated parameter
```

Comment:
There should not be a level 3 and level 4 header, as TOR102 is just one item.
See TOR003 for example.

The title should be specific for the issue, not generic.